src/edgekv/ekv-cli-main.ts

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env node
+import * as envUtils from '../utils/env-utils';
+import * as cliUtils from '../utils/cli-utils';
+import * as kvCliHandler from './ekv-handler';
+import * as edgeWorkersClientSvc from '../edgeworkers/client-manager';
+import * as httpEdge from '../cli-httpRequest';
+import * as pkginfo from '../../package.json';
+const commander = require('commander');
+const program = new commander.Command();
+const copywrite = '\nCopyright (c) 2019-2020 Akamai Technologies, Inc. Licensed under Apache 2 license.\nYour use of Akamai\'s products and services is subject to the terms and provisions outlined in Akamai\'s legal policies.\nVisit http://github.com/akamai/cli-edgeworkers for detailed documentation';
+program
+  .version(pkginfo.version)
+  .description(pkginfo.description)
+  .option('--debug', 'Show debug information.')
+  .option('--edgerc <path>', 'Use edgerc file for authentication.')
+  .option('--section <name>', 'Use this section in edgerc file that contains the credential set.')
+  // .option('--json [path]', 'Write command output to JSON file at given path, otherwise written to CLI cache directory')
```

Comment:
Any reason why the --json flag isn't honored by EKVCLI?  This is useful for customer that have automation that leverages our CLIs

Reply:
yes ! Not sure if you recall, we had a discussion regarding spinner earlier. We are planning to add --json support in the next release. have a ticket created for it as well. https://track.akamai.com/jira/browse/EDB-630

Reply:
Ah right - as long as it is coming later that's fine with me!